test/src/unit-capi-array_schema.cc

```diff
@@ -1276,6 +1276,9 @@ TEST_CASE_METHOD(
   rc = tiledb_array_schema_set_offsets_filter_list(
       ctx_, array_schema, filter_list);
   REQUIRE(rc == TILEDB_OK);
+  rc = tiledb_array_schema_set_validity_filter_list(
```

Comment:
I see some usage in unit tests for the offset filter counterpart... Could we add those API to the mix?

Reply:
I'm not sure I follow, do you mind to clarify? The offsets_filter setter is tested above, and the getter on 1320: https://github.com/TileDB-Inc/TileDB/pull/2798/files#diff-1fc513c5533bebab4dc78095c39f7ed86e5bea42ab9d24eddf829c9a70d71d66R1320